site/src/pages/tutorials/grpc/blog/define-service.mdx

```diff
@@ -106,6 +106,12 @@ Compile the `blog.proto` file to generate Java code.
 ./gradlew generateProto
 ```
 
+<Tip>
+
+  If you cannot execute the command with message `Task 'generateProto' not found in root project 'armeria-grpc'`, you need to configure the build.gradle by referring to [line/armeria-examples](https://github.com/line/armeria-examples/blob/master/tutorials/grpc/build.gradle).
```

Comment:
How about just
```
Please refer to [build.gradle](...) for generating code with [Protobuf Plugin for Gradle](https://github.com/google/protobuf-gradle-plugin).
```

Reply:
Thanks 🙇‍♂️ I will apply the changes you suggested.